automatic/1password/update.ps1

```diff
@@ -15,36 +14,50 @@ function global:au_SearchReplace {
   }
 }
 
-function global:au_AfterUpdate {
+function global:au_BeforeUpdate {
   . "$PSScriptRoot/update_helper.ps1"
-  if ($Latest.PackageName -eq '1password4') {
     removeDependencies ".\*.nuspec"
-  } else {
+}
+
+function global:au_AfterUpdate {
+  . "$PSScriptRoot/update_helper.ps1"
+  if ($Latest.PackageName -eq '1password') {
     addDependency ".\*.nuspec" 'dotnet4.6.1' '4.6.01055.20170308'
-  }
+  } 
 }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
   $streams = @{}
-  $res = @('OPW4';'format=zip$')
+  $res = @('OPWY';'OPW4';'OPW7')
```

Comment:
What is the `OPWY`?

Reply:
It is a shorten version of the code they use for beta `OPW7/Y`